test/utils.js

```diff
@@ -167,8 +164,15 @@ var TestUtils = {
                     pt[k] = Math.round(v * 10000) / 10000;
                 }
             }
-        })
-        .value();
+        });
+        
+        if (massageOptions.sort) {
+            massageOptions.sort.forEach(function(sortField) {
+                arr = _.sortBy(sortField);
```

Comment:
Good catch, the test massaging was making all the testing using the sort massage inaccurate. Fixed now.